lang/en/filter_imageopt.php

```diff
@@ -29,6 +29,8 @@
 $string['donotloadonvisible'] = 'No placeholding, load immediately';
 $string['loadonvisibilityall'] = 'All images';
 $string['loadonvisibilityafter'] = 'After {$a} image(s)';
+$string['lovsmallimage'] = 'Load when visible for small images';
```

Comment:
I'd consider changing this to:

Load when visible for ALL images

Reply:
The reason I think this might be better is that with the current string, someone might think that it will ONLY apply load when visible for small images. Using the word "all" is much clearer.